readme.md

```diff
@@ -20,12 +20,10 @@ It requires a key, which is provided by the Key module. To manage keys, visit `a
 After configuring the service, the service provides the ability to encrypt and decrypt.
 
 ### Encrypt
-
-`Drupal::service('encryption')->encrypt($string);`
+`$encryption_profile = \Drupal::service('entity.manager')->getStorage('encryption_profile')->load($instance_id);`
```

Comment:
I like putting this in, it's a bit more intuitive, as it shows what the second parameter is. We should consider adding in $string as well (I know it sounds silly).

I hope it doesn't confuse anyone, the code is not really commented or anything.